src/Abp/Auditing/AuditInfo.cs

```diff
@@ -42,6 +42,11 @@ public class AuditInfo
         /// </summary>
         public string Parameters { get; set; }
 
+        /// <summary>
+        ///     Return values.
```

Comment:
To be consistent with other annotation, please avoid adding additional whitespaces